### PR TITLE
Adding CONTRIBUTING.md file

### LINKED ISSUE #11: Adding CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by finziamit**
Adding CONTRIBUTING.md file
resolve #11

### CODE REVIEW

**Review by NitzanMordhai — APPROVED** [review]
LGTM

**Review by Anas-mhameed — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#Contributing
```
Comment (Shalevro2) [inline CONTRIBUTING.md]: I think you need to add a space after the hashtag for create a h1 header.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+welcome to the CONTRIBUTING.md file of �Smarticle� project. Please read this document thoroughly - follow it in all your interactions with the project.
+
+##Pre-Requirements
```
Comment (Shalevro2) [inline CONTRIBUTING.md]: Also here and in the rest of the file

**Review by Shalevro2 — CHANGES_REQUESTED** [review]
Add a space between hashtags and headers text.

File: CONTRIBUTING.md (hunk 2 above)
Comment (finziamit) [reply CONTRIBUTING.md]: done

**Review by Anas-mhameed — APPROVED** [review]

**Comment by avi-biton** [comment]
1 - Please add space after the # (like @Shalevro2 commented)
2 - When you open the file, you can see that there are no new lines.
To add new line wherever you want, you need to insert 2 spaces at the end of the line, for example:
`this is one line  `
`this is the second line`
3 - If you want to use dots for sections (like: `•	Giving and gracefully accepting constructive feedback`), use `*` instead of copying the `•`
So this line for example will look like: `*	Giving and gracefully accepting constructive feedback`

Note for you all, almost every IDE (pycharm for example) has the option of md file preview, so you can see the result while you are writing the md file.
Other way is the link that @Shalevro2 added to the slack: https://markdownlivepreview.com/
you can also use https://www.markdownguide.org/cheat-sheet/ as a cheatsheet
Or github guide: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Review by Shalevro2 — APPROVED** [review]
LGTM

**Review by Anas-mhameed — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,62 +1,86 @@
...
+## Enforcement Responsibilities  
+Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.  
+  
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.  
+  
+## Scope  
+This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.  
+  
+## Enforcement  
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at Smarticle@walla.co.il. All complaints will be reviewed and investigated promptly and fairly.
```
Comment (avi-biton) [inline CONTRIBUTING.md:55]: Is Smarticle@walla.co.il an actual mail? does it really exist?

**Pull request merged**